atomic.go

```diff
@@ -76,6 +77,21 @@ func (i *Int32) Swap(n int32) int32 {
 	return atomic.SwapInt32(&i.v, n)
 }
 
+// MarshalJSON allows for json encoding of the wrapped int32
```

Comment:
godoc comments should be full sentences ending in period. How about:

```suggestion
// MarshalJSON encodes the wrapped int32 into JSON.
```